src/unix/fs.c

```diff
@@ -66,6 +66,8 @@
 
 #define INIT(subtype)                                                         \
   do {                                                                        \
+    if (req == NULL)                                                          \
```

Comment:
Then `uv_fs_req_cleanup` should check if req is null and not do anything: https://github.com/libuv/libuv/blob/v1.x/src/unix/fs.c#L1452

Reply:
Same on Windows.